julia.html.markdown

```diff
@@ -780,9 +780,9 @@ code_native(square_area, (Float32,), syntax = :intel)
     #         nop     word ptr [rax + rax]
     # ;}
 
-code_native(square_area, (Float64,), syntax = :intel)
+codeNative(squareArea, (Float64,), syntax = :intel)
     #         .text
-    # ; Function square_area {
+    # ; Function squareArea {
```

Comment:
Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194